package.json

```diff
@@ -46,6 +47,7 @@
     "react-dom": "15.1.0",
     "react-draggable": "2.2.1",
     "react-tooltip": "3.0.13",
+    "sign-addon": "0.1.3",
```

Comment:
Not sure if we're using this dependency anywhere [yet]. Only other reference to it that I see in this PR is my stupid comment above: https://github.com/meandavejustice/min-vid/pull/162/files#r77084681